### ISSUE TITLE
Pre-work Checklist: Arpita Pandya

### ISSUE THREAD

**Issue by arpitapandya** [hackforla/website#3152]
### Prerequisite
We are looking forward to having you on our team. Please make sure to attend the general Hack for LA onboarding to get the process started https://meetup.com/hackforla/events

### Overview
As a new developer on the HfLA website team, fill in the following fields as you complete each onboarding item.

### Action Items
- [x] Add yourself to the #hfla-site and #hfla-site-pr Slack channels
- [x] Share your Gmail address with the team lead so you can be added to the Google Drive
- [x] Also, confirm with the team lead that they have added you to the Zoom meeting calendar invites
- [x] (Once added to the Drive) Add yourself to the [team roster](https://docs.google.com/spreadsheets/d/11u71eT-rZTKvVP8Yj_1rKxf2V45GCaFz4AXA7tS_asM/edit#gid=0)
- [x] After you have finished adding yourself to the roster, let a team lead know you have added yourself to the roster and would like to be added to the `website-write` and `website` teams on GitHub
- [x] Once added to the website-write team:
  - [x] Self Assign this issue (gear in right side panel)
  - [x] Add the "role: front end" or "role: back end" or both label(s) to this issue and remove the "role missing" label
  - [x] Add this issue to the Project Board under the Projects section (gear in right side panel)
- [x] Signup for a [Figma](https://Figma.com) account
- [x] Attend weekly team meetings:
  - [x] Developer (front-end/back-end) weekly team meeting, Tuesdays 7-8pm PST
  - [x] (Optional) Office Hours, Thursdays 7-8pm PST
  - [x] All team meeting (UX, Development, Product), Sunday 10am-12pm PST
- [x] Note regarding weekly team meeting requirements: All website team members are required to attend at least 1 team meeting in a week (held on Tuesdays, Thursdays and Sundays). In case, you are unable in any given week, you should reach out to the tech leadership team. Exceptions to this requirement may be provided on a case-by-case basis. Also, please let the tech leadership team know (through a slack message in the hfla-site channel as well as an @ mention in a comment of the issue that you would be working on) if you are planning to take a week off or a longer vacation.
- [x] Complete steps 1.1 - 1.6 in [Part 1: Setting up the development environment within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-1-setting-up-the-development-environment)
  - [x] OPTIONAL: If you run into any issues, use [4.1 How do I ask for help within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#41-what-do-i-do-if-i-need-help) as a visual example to get a hold of our [HfLA website leadership team](https://github.com/hackforla/website/projects/7#card-69730135). That way we can help you resolve any set up issues immediately.
- [x] Read section 2.1 - 2.4 in [Part 2: How the Website team works with GitHub issues within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#part-2-how-the-website-team-works-with-github-issues)
  - [x] Follow the steps in section [2.3 Where can I find GitHub issues to work on?](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#23-where-can-i-find-github-issues-to-work-on) to assign yourself your first issue under the Project Board's Prioritized Backlog column and use [2.7 Working on a Issue within Contributing.md](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#27-working-on-an-issue) to start working on your issue
  - [x] Read [2.6 What to do when you need to stop mid issue](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md#26-what-to-do-when-you-need-to-stop-mid-issue)
- [x] Once you take a good first issue, you will provide estimates and progress reports. Also, once you finish providing your "Estimate" action item below, please answer the question in the "Time spent so far" action item (also below).
   - [x] Estimate: Copy the below and put it in the "good first issue" that you picked.
 Check this box when you have completed this task so that we can identify if you understood the instructions and know what to do on all subsequent issues upon assignment.
    ```
   Availability for this week:
   My estimated ETA for completing this issue:
    ```
   - [x] Time spent so far: Copy the question below into a new comment below and answer it. This is just to get feedback on how long it took you to get to this point on this pre-work. There is no right or wrong answers. There is no judgement. It is ok if you take a long time or if you do it really fast or at any pace.
    ```
    How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?
    ```
   - [x] Progress Reports: Copy the below and put it in the issue once you have been assigned to the issue at least 5 days (we check weekly on Fridays), or sooner if you have something to report.  If you finish this issue before 5 days are reached, Yeah!!, do it on your next issue.  This update should be done every week for every issue.  The checkbox here is meant for us to see if you understood the instructions when you end up doing your first weekly progress update.
    ```
    Provide Update
    1. Progress
    2. Blockers
    3. Availability
    4. ETA
    ```
- [x] Read sections 3.1.a - 3.1.c in [3.1 How to make a pull request](https://github.com/hackforla/website/blob/2e3c45fede99e61f479548ccca3f0dda83e351db/CONTRIBUTING.md#31-how-to-make-a-pull-request) to learn more about how to make a pull request (PR) for the issue that you are working on and how to make changes to your PR if changes are requested by the reviewer
- [x] Additional notes:
   - [x] Please don't forget to review pull requests whenever you are able to.  The [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests) guide will teach you how to review pull requests.
   - [x] Please work on only one issue at a time and wait until your pull request is merged before picking up another issue.
- [x] Read and understand how we progress through issues. Then, you can check this off.
Progress through issues with increasing complexity in the following order:
  - Good first issue (one per person)
  - Good second issue (one per person)
  - Small (one per person, with some exceptions, see below)
  - Medium (you can work on more than one medium issue, but only one at a time)
  - Large (you can work on more than one large issue, but only one at a time)
    - The reasons for this progression are:
      - The issues start out as being prescriptive and become less so as you gain more experience by working through increasingly complex issues.
      - We are trying to teach you the team methodology through the issues themselves.
      - It ensures you understand what we expect and the quality of contributions.
    - You can work on back-to-back small issues if it meets the following criteria:
      - You are learning something new and need to work on an issue of a lesser complexity
      - Special request by a lead or pm
- [x] Go familiarize yourself with the [Hack for LA Design System page in Figma](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website?node-id=3464%3A3)  (where you can see components and their SCSS classes)
- [x] Once all tasks are completed, close this issue.  This issue will remain open until you get through your first progress report.  Once you have done that, we are confident you know how to keep the momentum going on your issue and keep your team informed.

### What should I do if I have a question about an issue I'm working on, and I haven't gotten a response yet?
- First, you should post the question or blocker as a comment on your assigned issue, so it can be easily referred to in the next bullet points
- Then, add the issue to the "Development team meeting discussion items" column of the Project Board so that it can be addressed in the next development meeting
- You may also add the label "Status: Help Wanted" so other developers can see it and potentially help answer your question
- Lastly, you can post your question in the hfla slack channel and link the issue you're working on, so other volunteers can see and respond

### Resources/Instructions
- [Contributing.md - Hack for LA](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)
- [GitHub Project Board - Hack for LA](https://github.com/hackforla/website/projects/7)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Front End](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+front+end%22)
- [GitHub Project Board Prioritized Backlog highlighted and sorted by Back End/DevOps](https://github.com/hackforla/website/projects/7?card_filter_query=label%3A%22role%3A+back+end%2Fdevops%22)
- [Figma - Hack for LA](https://www.figma.com/file/0RRPy1Ph7HafI3qOITg0Mr/Hack-for-LA-Website)
- [Google Drive - Hack for LA website team](https://drive.google.com/drive/folders/1p76K0FgfiAWeIIEyoyJ_Iik8FVj8cBjT?usp=sharing)
- [Agenda / Notes - Dev Team Tuesday meeting](https://github.com/hackforla/website/issues/2010)
- [Agenda / Notes - All Team meeting](https://github.com/hackforla/website/issues/2027)
- [How to review Pull Requests](https://github.com/hackforla/website/wiki/How-to-review-pull-requests)

**Comment by SAUMILDHANKAR** [comment]
@arpitapandya Just wanted to follow up on your progress with this issue. Please check off the action items above as you go along. Also, please self assign this issue. Thank you.

**Comment by SAUMILDHANKAR** [comment]
@arpitapandya Good job on making progress with your prework issue. Looks like your dev environment is all setup. Please pick up your good first issue next. Thank you.

**Comment by ExperimentsInHonesty** [comment]
@arpitapandya We did not hear from you last week after we reached out and its been 12 days since you joined the team, please write back as a comment on this issue 
1. Progress: "What is the current status of your getting set up? What have you completed and what is left to do?"
2. Blockers: "Difficulties or errors encountered."
3. Availability: "How much time will you have this week to work on this issue?"
4. ETA: "When do you expect this issue to be completed?"
5. Pictures (if necessary): "Add any pictures that will help illustrate what you are working on."

If we do not hear from you in the next week, you will be removed from the project, until you are ready to come back.

**Comment by arpitapandya** [comment]
@ExperimentsInHonesty @SAUMILDHANKAR I have picked Good first issue https://github.com/hackforla/website/issues/2878 and working on it. No blockers. I am expecting to complete before end of next week. I am available min 10hrs a week.

**Comment by tamara-snyder** [comment]
Hi @arpitapandya, I am reopening your Pre-Work because you still need to report your time spent on the Pre-Work Checklist in a comment.

I have unchecked the item for the hours spent so far. Please check it off again once this is done.

**Comment by arpitapandya** [comment]
How many hours did it take you to finish the pre-work up to and including adding your initial ETA and availability for your good first issue, including attending your first meetings?:
Time spent: 6hrs to complete
Availability: 6hrs for next week
Blockers: None, I am submitting late as my laptop was in service few weeks.

**Comment by HackforLABot** [comment]
## Activity Log: arpitapandya
### Repo: https://github.com/hackforla/website

#####  ⚠ Important note: The bot updates this comment automatically - do not edit

- arpitapandya opened: https://github.com/hackforla/website/issues/3151 at 05/16/2022, 08:05 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/issues/3152 at 05/16/2022, 08:13 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/issues/3155 at 05/17/2022, 08:14 PM PDT
- arpitapandya closed as completed: https://github.com/hackforla/website/issues/3155#event-6628886557 at 05/17/2022, 08:24 PM PDT
- arpitapandya closed as completed: https://github.com/hackforla/website/issues/3151#event-6628887594 at 05/17/2022, 08:24 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3152#issuecomment-1135330178 at 05/24/2022, 06:55 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/2878#issuecomment-1049097964 at 05/28/2022, 03:24 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3152#issuecomment-1140491243 at 05/29/2022, 10:23 AM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3185 at 05/31/2022, 08:37 AM PDT
- arpitapandya closed as completed: https://github.com/hackforla/website/issues/3152#event-6714149307 at 05/31/2022, 06:23 PM PDT
- arpitapandya closed by PR 3185: https://github.com/hackforla/website/issues/2878#event-6714189188 at 05/31/2022, 06:32 PM PDT
- arpitapandya reopened: https://github.com/hackforla/website/issues/3152#event-6714149307 at 06/04/2022, 10:27 AM PDT
- arpitapandya closed as completed: https://github.com/hackforla/website/issues/3152#event-6742261294 at 06/04/2022, 10:28 AM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3216#issuecomment-1145860440 at 06/04/2022, 10:43 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3216#issuecomment-1146679677 at 06/04/2022, 01:19 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3221 at 06/05/2022, 09:23 AM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3210#issuecomment-1145850932 at 06/05/2022, 09:59 AM PDT
- arpitapandya unassigned: https://github.com/hackforla/website/issues/3210#issuecomment-1146889828 at 06/05/2022, 02:54 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3216#issuecomment-1147538130 at 06/06/2022, 07:53 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3221#event-6755927925 at 06/06/2022, 01:25 PM PDT
- arpitapandya reopened: https://github.com/hackforla/website/pull/3221#event-6755927925 at 06/06/2022, 01:25 PM PDT
- arpitapandya issue closed by PR 3221: https://github.com/hackforla/website/issues/3216#event-6757146775 at 06/06/2022, 06:20 PM PDT
- arpitapandya PR was merged: https://github.com/hackforla/website/pull/3221#event-6757146781 at 06/06/2022, 06:20 PM PDT
- arpitapandya PR was merged: https://github.com/hackforla/website/pull/3185#event-6757189268 at 06/06/2022, 06:34 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3238#issuecomment-1153226022 at 06/14/2022, 01:54 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3238#issuecomment-1155707347 at 06/14/2022, 01:59 PM PDT
- arpitapandya unassigned: https://github.com/hackforla/website/issues/3238#issuecomment-1155803187 at 06/14/2022, 04:26 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/2953#issuecomment-1133414539 at 06/19/2022, 05:42 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2953#issuecomment-1159848519 at 06/19/2022, 05:43 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3152#issuecomment-1185593640 at 07/15/2022, 07:19 AM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3342#issuecomment-1179635333 at 07/15/2022, 07:32 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3342#issuecomment-1185607241 at 07/15/2022, 07:34 AM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3374 at 07/15/2022, 12:15 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3375 at 07/16/2022, 03:17 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2953#issuecomment-1186321107 at 07/16/2022, 04:07 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2953#issuecomment-1186321596 at 07/16/2022, 04:09 PM PDT
- arpitapandya closed as completed: https://github.com/hackforla/website/issues/3152#event-7006648755 at 07/16/2022, 04:17 PM PDT
- arpitapandya issue closed by PR 3375: https://github.com/hackforla/website/issues/2953#event-7013231672 at 07/18/2022, 08:39 AM PDT
- arpitapandya PR was merged: https://github.com/hackforla/website/pull/3375#event-7013231702 at 07/18/2022, 08:39 AM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3385 at 07/18/2022, 11:53 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3342#issuecomment-1188162260 at 07/18/2022, 12:17 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3296#issuecomment-1164984752 at 07/18/2022, 12:31 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3296#issuecomment-1188215836 at 07/18/2022, 12:32 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3391 at 07/19/2022, 09:48 AM PDT
- arpitapandya closed: https://github.com/hackforla/website/pull/3391#event-7022397733 at 07/19/2022, 09:49 AM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3392 at 07/19/2022, 11:56 AM PDT
- arpitapandya closed: https://github.com/hackforla/website/pull/3392#event-7023231694 at 07/19/2022, 12:05 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3393 at 07/19/2022, 12:08 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3296#issuecomment-1189460232 at 07/19/2022, 12:16 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3385#issuecomment-1189477833 at 07/19/2022, 12:37 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3385#issuecomment-1189542751 at 07/19/2022, 01:57 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3337#issuecomment-1177011694 at 07/19/2022, 02:10 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3337#issuecomment-1189553922 at 07/19/2022, 02:11 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3394 at 07/19/2022, 02:32 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3337#issuecomment-1189573575 at 07/19/2022, 02:34 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3385#issuecomment-1189608220 at 07/19/2022, 03:20 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3393#issuecomment-1190611368 at 07/20/2022, 11:23 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3393#issuecomment-1190868544 at 07/20/2022, 04:28 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3393#issuecomment-1190869596 at 07/20/2022, 04:30 PM PDT
- arpitapandya PR was merged: https://github.com/hackforla/website/pull/3394#event-7040384446 at 07/21/2022, 10:56 AM PDT
- arpitapandya PR was merged: https://github.com/hackforla/website/pull/3393#event-7047132854 at 07/22/2022, 07:51 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3385#issuecomment-1192899931 at 07/22/2022, 01:09 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3425#issuecomment-1193417394 at 07/24/2022, 04:31 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3425#pullrequestreview-1049775930 at 07/25/2022, 09:53 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3426#issuecomment-1194348326 at 07/25/2022, 09:54 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3426#issuecomment-1194549494 at 07/25/2022, 12:52 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3427#issuecomment-1194553558 at 07/25/2022, 12:56 PM PDT
- arpitapandya PR was merged: https://github.com/hackforla/website/pull/3385#event-7059399237 at 07/25/2022, 01:07 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3427#pullrequestreview-1051401975 at 07/26/2022, 10:54 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3429#issuecomment-1195816389 at 07/26/2022, 11:09 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3429#issuecomment-1196139826 at 07/26/2022, 05:31 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3426#issuecomment-1196141745 at 07/26/2022, 05:35 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/2626#issuecomment-1001077082 at 07/26/2022, 07:50 PM PDT
- arpitapandya closed by PR 3385: https://github.com/hackforla/website/issues/3342#event-7069336597 at 07/26/2022, 07:53 PM PDT
- arpitapandya closed by PR 3394: https://github.com/hackforla/website/issues/3337#event-7069339382 at 07/26/2022, 07:54 PM PDT
- arpitapandya closed by PR 3393: https://github.com/hackforla/website/issues/3296#event-7069342022 at 07/26/2022, 07:55 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3429#issuecomment-1197486141 at 07/27/2022, 04:31 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3429#issuecomment-1197488981 at 07/27/2022, 04:36 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3435#issuecomment-1197540367 at 07/27/2022, 06:14 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3435#pullrequestreview-1053384607 at 07/27/2022, 06:29 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3436#issuecomment-1198265615 at 07/28/2022, 07:58 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3436#pullrequestreview-1054290191 at 07/28/2022, 08:05 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3436#pullrequestreview-1054583664 at 07/28/2022, 11:37 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3436#pullrequestreview-1054584189 at 07/28/2022, 11:38 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3436#pullrequestreview-1054605483 at 07/28/2022, 11:57 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3436#issuecomment-1198587100 at 07/28/2022, 01:12 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3439#issuecomment-1198592615 at 07/28/2022, 01:19 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3439#pullrequestreview-1054699478 at 07/28/2022, 01:26 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3431#issuecomment-1198611620 at 07/28/2022, 01:41 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3441#issuecomment-1198845935 at 07/28/2022, 08:36 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3442#issuecomment-1199386572 at 07/29/2022, 07:25 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3442#pullrequestreview-1055666268 at 07/29/2022, 07:37 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3443#issuecomment-1199435624 at 07/29/2022, 07:45 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3443#pullrequestreview-1055696600 at 07/29/2022, 08:00 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3432#issuecomment-1199574118 at 07/29/2022, 08:43 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3432#pullrequestreview-1055813574 at 07/29/2022, 09:32 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3444#issuecomment-1200158900 at 07/30/2022, 06:33 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3444#pullrequestreview-1056466053 at 07/30/2022, 11:13 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3444#issuecomment-1200306297 at 07/30/2022, 03:40 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3436#issuecomment-1200310515 at 07/30/2022, 04:12 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3436#issuecomment-1200311379 at 07/30/2022, 04:19 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3436#issuecomment-1203394978 at 08/02/2022, 06:56 PM PDT
- arpitapandya unassigned: https://github.com/hackforla/website/issues/2626#issuecomment-1001077082 at 08/03/2022, 06:57 AM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3440#issuecomment-1198689313 at 08/03/2022, 02:02 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3440#issuecomment-1204496085 at 08/03/2022, 02:20 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3450 at 08/03/2022, 08:28 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3440#issuecomment-1204719722 at 08/03/2022, 08:40 PM PDT
- arpitapandya issue closed by PR 3450: https://github.com/hackforla/website/issues/3440#event-7132600072 at 08/04/2022, 07:33 PM PDT
- arpitapandya PR was merged: https://github.com/hackforla/website/pull/3450#event-7132600006 at 08/04/2022, 07:33 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3450#issuecomment-1205975588 at 08/04/2022, 07:34 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/2459#issuecomment-1065966125 at 08/04/2022, 08:29 PM PDT
- arpitapandya unassigned: https://github.com/hackforla/website/issues/2459#issuecomment-1065966125 at 08/04/2022, 08:33 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/2299#issuecomment-1190501160 at 08/04/2022, 08:36 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2299#issuecomment-1206007196 at 08/04/2022, 08:38 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3451 at 08/04/2022, 11:09 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2299#issuecomment-1206082772 at 08/04/2022, 11:14 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3436#issuecomment-1206090406 at 08/04/2022, 11:27 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3455#issuecomment-1208847032 at 08/08/2022, 07:39 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3457#issuecomment-1210054846 at 08/09/2022, 06:38 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3454#issuecomment-1210055059 at 08/09/2022, 06:39 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3454#pullrequestreview-1067561652 at 08/09/2022, 06:57 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3452#issuecomment-1210077343 at 08/09/2022, 07:27 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3457#pullrequestreview-1068876276 at 08/10/2022, 01:14 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3436#issuecomment-1211310905 at 08/10/2022, 02:43 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3451#issuecomment-1211532682 at 08/10/2022, 09:09 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3436#issuecomment-1211912712 at 08/11/2022, 05:18 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3462#issuecomment-1212692423 at 08/11/2022, 08:23 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3462#pullrequestreview-1070563055 at 08/11/2022, 08:38 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3452#pullrequestreview-1071996313 at 08/13/2022, 01:52 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3451#issuecomment-1214245305 at 08/13/2022, 04:22 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3468#issuecomment-1214444180 at 08/14/2022, 01:32 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3468#issuecomment-1214622334 at 08/14/2022, 10:06 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3472 at 08/17/2022, 11:59 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3468#pullrequestreview-1076321309 at 08/17/2022, 12:52 PM PDT
- arpitapandya closed: https://github.com/hackforla/website/pull/3472#event-7208839527 at 08/17/2022, 12:53 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3475#issuecomment-1220227804 at 08/18/2022, 09:10 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3475#pullrequestreview-1078253309 at 08/18/2022, 09:12 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2299#issuecomment-1229508544 at 08/28/2022, 10:03 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3490#event-7274007582 at 08/28/2022, 02:12 PM PDT
- arpitapandya closed by PR 3490: https://github.com/hackforla/website/issues/2205#event-7274007592 at 08/28/2022, 02:12 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3451#issuecomment-1229565187 at 08/28/2022, 03:15 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3483#issuecomment-1229568108 at 08/28/2022, 03:33 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3483#pullrequestreview-1087924443 at 08/28/2022, 04:03 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3483#event-7274130745 at 08/28/2022, 04:18 PM PDT
- arpitapandya closed by PR 3483: https://github.com/hackforla/website/issues/2172#event-7274130772 at 08/28/2022, 04:18 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3451#issuecomment-1230379918 at 08/29/2022, 07:20 AM PDT
- arpitapandya closed: https://github.com/hackforla/website/pull/3451#event-7279221249 at 08/29/2022, 07:20 AM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3496 at 08/29/2022, 07:58 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3497#issuecomment-1230899030 at 08/29/2022, 02:49 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3497#pullrequestreview-1089405052 at 08/29/2022, 03:03 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3497#event-7282462531 at 08/29/2022, 03:03 PM PDT
- arpitapandya closed by PR 3497: https://github.com/hackforla/website/issues/2199#event-7282462601 at 08/29/2022, 03:03 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3513#issuecomment-1233545970 at 08/31/2022, 04:23 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3513#pullrequestreview-1092615261 at 08/31/2022, 04:31 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3513#event-7300737308 at 08/31/2022, 05:39 PM PDT
- arpitapandya closed by PR 3513: https://github.com/hackforla/website/issues/2822#event-7300737386 at 08/31/2022, 05:39 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3514#issuecomment-1234784508 at 09/01/2022, 02:07 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3514#pullrequestreview-1094063750 at 09/01/2022, 02:10 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3514#event-7309935000 at 09/01/2022, 07:53 PM PDT
- arpitapandya closed by PR 3514: https://github.com/hackforla/website/issues/3101#event-7309935083 at 09/01/2022, 07:53 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3496#event-7315474606 at 09/02/2022, 12:47 PM PDT
- arpitapandya closed by PR 3496: https://github.com/hackforla/website/issues/2299#event-7315474661 at 09/02/2022, 12:47 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/2249#issuecomment-917485924 at 09/02/2022, 12:57 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1235853923 at 09/02/2022, 01:01 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3516#event-7316202636 at 09/02/2022, 04:20 PM PDT
- arpitapandya closed by PR 3516: https://github.com/hackforla/website/issues/2825#event-7316202671 at 09/02/2022, 04:20 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3517#issuecomment-1236020227 at 09/02/2022, 06:31 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3515#issuecomment-1236141705 at 09/03/2022, 08:19 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3515#pullrequestreview-1095523721 at 09/03/2022, 08:23 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3515#event-7317225546 at 09/03/2022, 08:27 AM PDT
- arpitapandya closed by PR 3515: https://github.com/hackforla/website/issues/2831#event-7317225581 at 09/03/2022, 08:27 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3517#issuecomment-1236150716 at 09/03/2022, 09:03 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3517#issuecomment-1236162584 at 09/03/2022, 09:46 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3517#pullrequestreview-1095541458 at 09/03/2022, 09:49 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3517#event-7318476335 at 09/04/2022, 06:34 AM PDT
- arpitapandya closed by PR 3517: https://github.com/hackforla/website/issues/3418#event-7318793922 at 09/04/2022, 11:32 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3518#issuecomment-1237577513 at 09/05/2022, 06:56 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3518#pullrequestreview-1096850453 at 09/05/2022, 06:58 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3518#event-7326306270 at 09/05/2022, 06:59 PM PDT
- arpitapandya closed by PR 3518: https://github.com/hackforla/website/issues/3198#event-7326306310 at 09/05/2022, 06:59 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3519#pullrequestreview-1096856544 at 09/05/2022, 07:12 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3519#event-7326375702 at 09/05/2022, 07:22 PM PDT
- arpitapandya closed by PR 3519: https://github.com/hackforla/website/issues/3097#event-7326375749 at 09/05/2022, 07:22 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3431#issuecomment-1237603839 at 09/05/2022, 07:56 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3431#pullrequestreview-1096880628 at 09/05/2022, 08:04 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3431#issuecomment-1238198033 at 09/06/2022, 07:03 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3431#pullrequestreview-1097658473 at 09/06/2022, 07:05 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3519#issuecomment-1238740760 at 09/06/2022, 04:15 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3423#issuecomment-1193030715 at 09/11/2022, 08:59 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3423#issuecomment-1242993539 at 09/11/2022, 08:59 AM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3534 at 09/11/2022, 09:57 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3476#issuecomment-1244796228 at 09/12/2022, 06:47 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3254#issuecomment-1245462286 at 09/13/2022, 07:04 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3534#event-7380484336 at 09/13/2022, 03:28 PM PDT
- arpitapandya closed by PR 3534: https://github.com/hackforla/website/issues/3423#event-7389389285 at 09/14/2022, 01:39 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1247289656 at 09/14/2022, 01:51 PM PDT
- arpitapandya unassigned: https://github.com/hackforla/website/issues/2249#issuecomment-1247289656 at 09/16/2022, 01:01 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3557#issuecomment-1249759311 at 09/16/2022, 01:06 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3557#pullrequestreview-1111172861 at 09/16/2022, 01:10 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3557#event-7405853110 at 09/16/2022, 01:11 PM PDT
- arpitapandya closed by PR 3557: https://github.com/hackforla/website/issues/2828#event-7405853546 at 09/16/2022, 01:11 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3558#issuecomment-1249772116 at 09/16/2022, 01:13 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2776#issuecomment-1249782405 at 09/16/2022, 01:27 PM PDT
- arpitapandya closed as completed: https://github.com/hackforla/website/issues/2776#event-7405956590 at 09/16/2022, 01:27 PM PDT
- arpitapandya closed by PR 3468: https://github.com/hackforla/website/issues/3295#event-7408296267 at 09/17/2022, 06:29 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3558#pullrequestreview-1111482649 at 09/17/2022, 06:43 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3558#issuecomment-1250172516 at 09/17/2022, 06:47 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3561#issuecomment-1250441098 at 09/18/2022, 06:13 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3561#pullrequestreview-1111630043 at 09/18/2022, 06:34 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3564#issuecomment-1251686748 at 09/19/2022, 04:52 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3566#issuecomment-1253785240 at 09/21/2022, 07:21 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3566#pullrequestreview-1115543921 at 09/21/2022, 07:32 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3566#event-7431572779 at 09/21/2022, 07:34 AM PDT
- arpitapandya closed by PR 3566: https://github.com/hackforla/website/issues/3106#event-7431573125 at 09/21/2022, 07:34 AM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/2668#issuecomment-1228152247 at 09/21/2022, 07:52 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2668#issuecomment-1253828938 at 09/21/2022, 07:54 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2249#issuecomment-1253834916 at 09/21/2022, 07:58 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3564#issuecomment-1253856721 at 09/21/2022, 08:15 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3561#pullrequestreview-1116139029 at 09/21/2022, 02:58 PM PDT
- arpitapandya closed by PR 3561: https://github.com/hackforla/website/issues/3100#event-7434584103 at 09/21/2022, 02:59 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3561#event-7434583990 at 09/21/2022, 02:59 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3573#event-7434714127 at 09/21/2022, 03:26 PM PDT
- arpitapandya closed by PR 3573: https://github.com/hackforla/website/issues/2834#event-7434714225 at 09/21/2022, 03:26 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3575 at 09/22/2022, 06:22 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3564#issuecomment-1256462551 at 09/23/2022, 10:17 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3564#pullrequestreview-1118808049 at 09/23/2022, 10:23 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3564#event-7449346733 at 09/23/2022, 10:52 AM PDT
- arpitapandya closed by PR 3564: https://github.com/hackforla/website/issues/1750#event-7449346835 at 09/23/2022, 10:52 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3574#issuecomment-1256510488 at 09/23/2022, 10:54 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3574#pullrequestreview-1118864837 at 09/23/2022, 10:56 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3574#event-7451018709 at 09/23/2022, 05:28 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3576#issuecomment-1256816462 at 09/23/2022, 05:33 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3576#pullrequestreview-1119126744 at 09/23/2022, 05:33 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3576#event-7451027780 at 09/23/2022, 05:34 PM PDT
- arpitapandya closed by PR 3576: https://github.com/hackforla/website/issues/2842#event-7451027811 at 09/23/2022, 05:34 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3577#issuecomment-1256818066 at 09/23/2022, 05:40 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3577#pullrequestreview-1119128310 at 09/23/2022, 05:43 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3577#event-7451040893 at 09/23/2022, 05:44 PM PDT
- arpitapandya closed by PR 3577: https://github.com/hackforla/website/issues/3099#event-7451040904 at 09/23/2022, 05:44 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3098#issuecomment-1256819950 at 09/23/2022, 05:49 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3578#event-7451053432 at 09/23/2022, 05:53 PM PDT
- arpitapandya closed by PR 3578: https://github.com/hackforla/website/issues/3098#event-7451053449 at 09/23/2022, 05:53 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3581#pullrequestreview-1120372107 at 09/26/2022, 07:47 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3579#event-7459841963 at 09/26/2022, 08:05 AM PDT
- arpitapandya closed by PR 3579: https://github.com/hackforla/website/issues/3129#event-7459842265 at 09/26/2022, 08:05 AM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3583 at 09/26/2022, 11:55 AM PDT
- arpitapandya closed: https://github.com/hackforla/website/pull/3583#event-7461641942 at 09/26/2022, 11:56 AM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3584 at 09/26/2022, 01:14 PM PDT
- arpitapandya closed: https://github.com/hackforla/website/pull/3584#event-7462659146 at 09/26/2022, 02:23 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3581#pullrequestreview-1120963177 at 09/26/2022, 03:05 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3582#issuecomment-1259729704 at 09/27/2022, 09:12 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3582#pullrequestreview-1122367892 at 09/27/2022, 10:01 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3582#event-7473343483 at 09/27/2022, 06:02 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3585#event-7473353495 at 09/27/2022, 06:05 PM PDT
- arpitapandya closed by PR 3585: https://github.com/hackforla/website/issues/2925#event-7473353558 at 09/27/2022, 06:05 PM PDT
- arpitapandya closed by PR 3581: https://github.com/hackforla/website/issues/3489#event-7473359091 at 09/27/2022, 06:06 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3581#event-7473359028 at 09/27/2022, 06:06 PM PDT
- arpitapandya closed: https://github.com/hackforla/website/pull/3575#event-7473542008 at 09/27/2022, 06:57 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3593#issuecomment-1263637218 at 09/30/2022, 07:18 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3593#issuecomment-1263820054 at 09/30/2022, 10:13 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3593#issuecomment-1264011288 at 09/30/2022, 01:48 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3593#pullrequestreview-1127348285 at 09/30/2022, 01:50 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3593#event-7498804032 at 09/30/2022, 02:36 PM PDT
- arpitapandya closed by PR 3593: https://github.com/hackforla/website/issues/2485#event-7498804089 at 09/30/2022, 02:36 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3589#issuecomment-1264054789 at 09/30/2022, 02:49 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3589#pullrequestreview-1127389447 at 09/30/2022, 02:50 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3589#event-7498864872 at 09/30/2022, 02:52 PM PDT
- arpitapandya closed by PR 3589: https://github.com/hackforla/website/issues/2954#event-7498864932 at 09/30/2022, 02:52 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3590#pullrequestreview-1127409744 at 09/30/2022, 03:32 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3590#event-7499029640 at 09/30/2022, 03:40 PM PDT
- arpitapandya closed by PR 3590: https://github.com/hackforla/website/issues/2836#event-7499029680 at 09/30/2022, 03:40 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3590#issuecomment-1264103211 at 09/30/2022, 03:44 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3594#issuecomment-1264427679 at 10/01/2022, 10:16 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3594#pullrequestreview-1127549598 at 10/01/2022, 10:23 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3586#issuecomment-1264449331 at 10/01/2022, 11:43 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3586#pullrequestreview-1127572286 at 10/01/2022, 03:26 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3595#issuecomment-1264510532 at 10/01/2022, 04:53 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3595#pullrequestreview-1127578993 at 10/01/2022, 05:42 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3595#event-7501258135 at 10/01/2022, 05:43 PM PDT
- arpitapandya closed by PR 3595: https://github.com/hackforla/website/issues/3562#event-7501258139 at 10/01/2022, 05:43 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3594#event-7501259931 at 10/01/2022, 05:45 PM PDT
- arpitapandya closed by PR 3594: https://github.com/hackforla/website/issues/3481#event-7501259941 at 10/01/2022, 05:45 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3586#issuecomment-1264518535 at 10/01/2022, 05:48 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3586#event-7501262214 at 10/01/2022, 05:48 PM PDT
- arpitapandya closed by PR 3586: https://github.com/hackforla/website/issues/3447#event-7501262221 at 10/01/2022, 05:48 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2668#issuecomment-1264691584 at 10/02/2022, 10:16 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3599#event-7546732083 at 10/07/2022, 02:37 PM PDT
- arpitapandya closed by PR 3599: https://github.com/hackforla/website/issues/3354#event-7546732147 at 10/07/2022, 02:37 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2355#issuecomment-1272422869 at 10/08/2022, 05:44 PM PDT
- arpitapandya closed by PR 3087: https://github.com/hackforla/website/issues/2355#event-7548678915 at 10/08/2022, 05:45 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3600#issuecomment-1272428566 at 10/08/2022, 06:36 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3447#issuecomment-1272603882 at 10/09/2022, 11:43 AM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3601 at 10/09/2022, 04:48 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3447#issuecomment-1272665955 at 10/09/2022, 05:29 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3600#pullrequestreview-1135412578 at 10/09/2022, 05:32 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3600#event-7550254365 at 10/09/2022, 06:18 PM PDT
- arpitapandya closed by PR 3600: https://github.com/hackforla/website/issues/2844#event-7550254434 at 10/09/2022, 06:18 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2286#issuecomment-1272692834 at 10/09/2022, 06:34 PM PDT
- arpitapandya PR was merged: https://github.com/hackforla/website/pull/3601#event-7555933563 at 10/10/2022, 10:20 AM PDT
- arpitapandya issue closed by PR 3586: https://github.com/hackforla/website/issues/3447#event-7555933662 at 10/10/2022, 10:20 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3615#issuecomment-1276965670 at 10/12/2022, 08:04 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3238#issuecomment-1278433610 at 10/13/2022, 08:33 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3238#issuecomment-1278434245 at 10/13/2022, 08:34 PM PDT
- arpitapandya closed as completed: https://github.com/hackforla/website/issues/3238#event-7586677015 at 10/13/2022, 08:34 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2668#issuecomment-1281477120 at 10/17/2022, 01:48 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3175#issuecomment-1283323031 at 10/18/2022, 07:39 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2210#issuecomment-1287548632 at 10/21/2022, 05:12 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3655#event-7667999890 at 10/25/2022, 05:06 PM PDT
- arpitapandya closed by PR 3655: https://github.com/hackforla/website/issues/3284#event-7668000060 at 10/25/2022, 05:06 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3657#event-7668007375 at 10/25/2022, 05:08 PM PDT
- arpitapandya closed by PR 3657: https://github.com/hackforla/website/issues/3205#event-7668007449 at 10/25/2022, 05:08 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3658#event-7668013587 at 10/25/2022, 05:10 PM PDT
- arpitapandya closed by PR 3658: https://github.com/hackforla/website/issues/2853#event-7668013633 at 10/25/2022, 05:10 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3659#event-7668021141 at 10/25/2022, 05:13 PM PDT
- arpitapandya closed by PR 3659: https://github.com/hackforla/website/issues/2857#event-7668021213 at 10/25/2022, 05:13 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3664#issuecomment-1292829445 at 10/26/2022, 06:00 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3664#issuecomment-1292831553 at 10/26/2022, 06:03 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/issues/3669 at 10/27/2022, 09:10 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3669 at 10/27/2022, 09:48 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/issues/3670 at 10/27/2022, 10:18 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3670 at 10/27/2022, 10:39 PM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3671 at 10/27/2022, 11:16 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2777#issuecomment-1295089805 at 10/28/2022, 07:45 AM PDT
- arpitapandya closed as completed: https://github.com/hackforla/website/issues/2777#event-7692384015 at 10/28/2022, 07:48 AM PDT
- arpitapandya closed as completed: https://github.com/hackforla/website/issues/3669#event-7692494249 at 10/28/2022, 08:02 AM PDT
- arpitapandya closed: https://github.com/hackforla/website/pull/3671#event-7692621650 at 10/28/2022, 08:19 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3667#issuecomment-1295367794 at 10/28/2022, 12:20 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3673#event-7698169080 at 10/29/2022, 08:14 PM PDT
- arpitapandya closed by PR 3673: https://github.com/hackforla/website/issues/3206#event-7698169103 at 10/29/2022, 08:14 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3652#event-7699613081 at 10/30/2022, 11:54 AM PDT
- arpitapandya closed by PR 3652: https://github.com/hackforla/website/issues/2438#event-7699613099 at 10/30/2022, 11:54 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3662#issuecomment-1296368287 at 10/30/2022, 03:44 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3662#pullrequestreview-1161206515 at 10/30/2022, 04:05 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3674#issuecomment-1296433691 at 10/30/2022, 06:42 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3674#pullrequestreview-1161252422 at 10/30/2022, 07:03 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3674#event-7700332095 at 10/30/2022, 07:04 PM PDT
- arpitapandya closed by PR 3674: https://github.com/hackforla/website/issues/2479#event-7700332161 at 10/30/2022, 07:04 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3676#pullrequestreview-1161257745 at 10/30/2022, 07:16 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3677#issuecomment-1297508977 at 10/31/2022, 11:36 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3677#pullrequestreview-1162365150 at 10/31/2022, 11:37 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3677#event-7707028521 at 10/31/2022, 11:38 AM PDT
- arpitapandya closed by PR 3677: https://github.com/hackforla/website/issues/3196#event-7707028728 at 10/31/2022, 11:38 AM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3675#issuecomment-1297512212 at 10/31/2022, 11:39 AM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3675#pullrequestreview-1162372680 at 10/31/2022, 11:43 AM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3675#event-7707075171 at 10/31/2022, 11:44 AM PDT
- arpitapandya closed by PR 3675: https://github.com/hackforla/website/issues/3278#event-7707075466 at 10/31/2022, 11:44 AM PDT
- arpitapandya opened: https://github.com/hackforla/website/pull/3680 at 10/31/2022, 12:56 PM PDT
- arpitapandya PR was merged: https://github.com/hackforla/website/pull/3680#event-7707754134 at 10/31/2022, 01:14 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/2853#issuecomment-1297644812 at 10/31/2022, 01:29 PM PDT
- arpitapandya reopened: https://github.com/hackforla/website/issues/2853#issuecomment-1297644812 at 10/31/2022, 01:29 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3681#issuecomment-1298029324 at 10/31/2022, 09:59 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3681#pullrequestreview-1162836673 at 10/31/2022, 10:12 PM PDT
- arpitapandya merged: https://github.com/hackforla/website/pull/3681#event-7710191838 at 10/31/2022, 10:13 PM PDT
- arpitapandya closed by PR 3681: https://github.com/hackforla/website/issues/3619#event-7710191937 at 10/31/2022, 10:13 PM PDT
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3687#pullrequestreview-1164393193 at 11/01/2022, 09:13 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3597#issuecomment-1299551555 at 11/01/2022, 09:31 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3687#issuecomment-1299571006 at 11/01/2022, 09:57 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/pull/3024#issuecomment-1301510691 at 11/02/2022, 05:10 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3641#issuecomment-1304058300 at 11/04/2022, 12:23 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3641#issuecomment-1304061730 at 11/04/2022, 12:26 PM PDT
- arpitapandya commented on: https://github.com/hackforla/website/issues/3641#issuecomment-1304064747 at 11/04/2022, 12:29 PM PDT
- arpitapandya assigned: https://github.com/hackforla/website/issues/3704 at 11/08/2022, 06:15 PM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3703#event-7786122481 at 11/10/2022, 10:47 AM PST
- arpitapandya closed by PR 3703: https://github.com/hackforla/website/issues/2859#event-7786122633 at 11/10/2022, 10:47 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3699#event-7786132153 at 11/10/2022, 10:48 AM PST
- arpitapandya closed by PR 3699: https://github.com/hackforla/website/issues/3207#event-7786132312 at 11/10/2022, 10:48 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3700#event-7786142990 at 11/10/2022, 10:50 AM PST
- arpitapandya closed by PR 3700: https://github.com/hackforla/website/issues/3208#event-7786143252 at 11/10/2022, 10:50 AM PST
- arpitapandya opened: https://github.com/hackforla/website/pull/3713 at 11/10/2022, 01:22 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/pull/3713#issuecomment-1310939323 at 11/10/2022, 01:41 PM PST
- arpitapandya closed: https://github.com/hackforla/website/pull/3713#event-7787273427 at 11/10/2022, 01:41 PM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3705#pullrequestreview-1176534782 at 11/10/2022, 01:44 PM PST
- arpitapandya reopened: https://github.com/hackforla/website/pull/3713#event-7787273427 at 11/10/2022, 02:37 PM PST
- arpitapandya opened: https://github.com/hackforla/website/pull/3714 at 11/10/2022, 06:10 PM PST
- arpitapandya PR was merged: https://github.com/hackforla/website/pull/3714#event-7788679852 at 11/10/2022, 07:17 PM PST
- arpitapandya opened: https://github.com/hackforla/website/pull/3715 at 11/10/2022, 07:52 PM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3705#pullrequestreview-1176779917 at 11/10/2022, 08:23 PM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3705#event-7788913511 at 11/10/2022, 08:24 PM PST
- arpitapandya closed by PR 3705: https://github.com/hackforla/website/issues/3449#event-7788913591 at 11/10/2022, 08:24 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/pull/3715#issuecomment-1311874739 at 11/11/2022, 07:53 AM PST
- arpitapandya closed: https://github.com/hackforla/website/pull/3715#event-7793166086 at 11/11/2022, 07:53 AM PST
- arpitapandya commented on: https://github.com/hackforla/website/pull/3713#issuecomment-1311878767 at 11/11/2022, 07:56 AM PST
- arpitapandya closed: https://github.com/hackforla/website/pull/3713#event-7793187728 at 11/11/2022, 07:56 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3696#event-7793206822 at 11/11/2022, 07:57 AM PST
- arpitapandya closed by PR 3696: https://github.com/hackforla/website/issues/3651#event-7793207171 at 11/11/2022, 07:57 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3702#event-7793222809 at 11/11/2022, 07:58 AM PST
- arpitapandya closed by PR 3702: https://github.com/hackforla/website/issues/3165#event-7793223171 at 11/11/2022, 07:58 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3718#event-7829138752 at 11/16/2022, 04:30 PM PST
- arpitapandya closed by PR 3718: https://github.com/hackforla/website/issues/2860#event-7829138872 at 11/16/2022, 04:30 PM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3717#event-7829148008 at 11/16/2022, 04:32 PM PST
- arpitapandya closed by PR 3717: https://github.com/hackforla/website/issues/2904#event-7829148120 at 11/16/2022, 04:32 PM PST
- arpitapandya closed by PR 3717: https://github.com/hackforla/website/issues/2904#event-7829148120 at 11/16/2022, 04:32 PM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3728#pullrequestreview-1184747046 at 11/17/2022, 09:27 AM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3728#pullrequestreview-1185019303 at 11/17/2022, 12:16 PM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3728#event-7837470791 at 11/17/2022, 12:17 PM PST
- arpitapandya closed by PR 3728: https://github.com/hackforla/website/issues/2873#event-7837471004 at 11/17/2022, 12:17 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/issues/2172#issuecomment-1328326558 at 11/27/2022, 11:52 AM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3754#pullrequestreview-1207091921 at 12/06/2022, 10:18 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3764#event-7969533967 at 12/06/2022, 10:23 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3762#event-7975681651 at 12/07/2022, 01:25 AM PST
- arpitapandya closed by PR 3497: https://github.com/hackforla/website/issues/2199#event-7975681885 at 12/07/2022, 01:25 AM PST
- arpitapandya commented on: https://github.com/hackforla/website/issues/3256#issuecomment-1347279715 at 12/12/2022, 12:42 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/issues/3697#issuecomment-1347295970 at 12/12/2022, 12:47 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/issues/3697#issuecomment-1347594392 at 12/12/2022, 04:59 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/issues/3697#issuecomment-1347624578 at 12/12/2022, 05:37 PM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3771#pullrequestreview-1214603633 at 12/12/2022, 05:44 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/pull/3771#issuecomment-1347653289 at 12/12/2022, 06:18 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/pull/3771#issuecomment-1349675036 at 12/13/2022, 12:53 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/issues/3697#issuecomment-1349799226 at 12/13/2022, 01:49 PM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3771#pullrequestreview-1216727182 at 12/13/2022, 07:22 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/issues/3697#issuecomment-1350341951 at 12/13/2022, 07:30 PM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3771#pullrequestreview-1216865135 at 12/13/2022, 10:32 PM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3771#event-8033411042 at 12/13/2022, 10:33 PM PST
- arpitapandya closed by PR 3771: https://github.com/hackforla/website/issues/3697#event-8033411170 at 12/13/2022, 10:33 PM PST
- arpitapandya commented on: https://github.com/hackforla/website/issues/3755#issuecomment-1351734938 at 12/14/2022, 08:31 AM PST
- arpitapandya commented on: https://github.com/hackforla/website/pull/3754#issuecomment-1358703433 at 12/19/2022, 05:14 PM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3795#event-8182898918 at 01/09/2023, 07:21 AM PST
- arpitapandya closed by PR 3795: https://github.com/hackforla/website/issues/2874#event-8182899276 at 01/09/2023, 07:21 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3806#event-8210903191 at 01/12/2023, 04:34 AM PST
- arpitapandya closed by PR 3806: https://github.com/hackforla/website/issues/3755#event-8210903395 at 01/12/2023, 04:34 AM PST
- arpitapandya commented on: https://github.com/hackforla/website/issues/2870#issuecomment-1380267739 at 01/12/2023, 04:38 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3804#event-8210939648 at 01/12/2023, 04:39 AM PST
- arpitapandya closed by PR 3804: https://github.com/hackforla/website/issues/2870#event-8210940025 at 01/12/2023, 04:39 AM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3808#pullrequestreview-1245533822 at 01/12/2023, 04:44 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3807#event-8211335969 at 01/12/2023, 05:25 AM PST
- arpitapandya closed by PR 3807: https://github.com/hackforla/website/issues/2861#event-8211336224 at 01/12/2023, 05:25 AM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3808#pullrequestreview-1247560999 at 01/13/2023, 04:29 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3808#event-8220398342 at 01/13/2023, 04:29 AM PST
- arpitapandya closed by PR 3808: https://github.com/hackforla/website/issues/3401#event-8220398566 at 01/13/2023, 04:29 AM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3812#pullrequestreview-1247570837 at 01/13/2023, 04:37 AM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3814#pullrequestreview-1249052516 at 01/14/2023, 03:51 PM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3817#event-8247174568 at 01/17/2023, 07:57 AM PST
- arpitapandya closed by PR 3817: https://github.com/hackforla/website/issues/3091#event-8247174752 at 01/17/2023, 07:57 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3815#event-8247186471 at 01/17/2023, 07:58 AM PST
- arpitapandya closed by PR 3815: https://github.com/hackforla/website/issues/2866#event-8247186711 at 01/17/2023, 07:58 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3810#event-8247198768 at 01/17/2023, 07:59 AM PST
- arpitapandya closed by PR 3810: https://github.com/hackforla/website/issues/3773#event-8247198981 at 01/17/2023, 07:59 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3832#event-8247207145 at 01/17/2023, 08:00 AM PST
- arpitapandya closed by PR 3832: https://github.com/hackforla/website/issues/2867#event-8247207463 at 01/17/2023, 08:00 AM PST
- arpitapandya commented on: https://github.com/hackforla/website/issues/2505#issuecomment-1386472117 at 01/17/2023, 08:30 PM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3842#event-8313338618 at 01/19/2023, 07:38 AM PST
- arpitapandya closed by PR 3842: https://github.com/hackforla/website/issues/2869#event-8313339233 at 01/19/2023, 07:38 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3834#event-8313372996 at 01/19/2023, 07:41 AM PST
- arpitapandya closed by PR 3834: https://github.com/hackforla/website/issues/3382#event-8313373368 at 01/19/2023, 07:41 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3838#event-8322133983 at 01/20/2023, 06:48 AM PST
- arpitapandya closed as completed: https://github.com/hackforla/website/issues/3415#event-8322144996 at 01/20/2023, 06:49 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3836#event-8322153040 at 01/20/2023, 06:50 AM PST
- arpitapandya closed by PR 3836: https://github.com/hackforla/website/issues/3402#event-8322153306 at 01/20/2023, 06:50 AM PST
- arpitapandya submitted review: https://github.com/hackforla/website/pull/3814#pullrequestreview-1263747936 at 01/20/2023, 06:53 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3814#event-8322185100 at 01/20/2023, 06:54 AM PST
- arpitapandya closed by PR 3814: https://github.com/hackforla/website/issues/2276#event-8322185418 at 01/20/2023, 06:54 AM PST
- arpitapandya commented on: https://github.com/hackforla/website/pull/3812#issuecomment-1399379774 at 01/21/2023, 05:33 PM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3864#event-8366877325 at 01/26/2023, 10:47 AM PST
- arpitapandya closed by PR 3864: https://github.com/hackforla/website/issues/3823#event-8366877522 at 01/26/2023, 10:47 AM PST
- arpitapandya merged: https://github.com/hackforla/website/pull/3865#event-8366881004 at 01/26/2023, 10:47 AM PST
- arpitapandya commented on: https://github.com/hackforla/website/pull/3898#issuecomment-1409236663 at 01/30/2023, 11:47 AM PST
- arpitapandya closed: https://github.com/hackforla/website/pull/3898#event-8391900219 at 01/30/2023, 11:47 AM PST
- arpitapandya commented on: https://github.com/hackforla/website/pull/3899#issuecomment-1409237551 at 01/30/2023, 11:48 AM PST
- arpitapandya closed: https://github.com/hackforla/website/pull/3899#event-8391906456 at 01/30/2023, 11:48 AM PST
- arpitapandya issue closed as completed: https://github.com/hackforla/website/issues/3670#event-8680332683 at 03/06/2023, 04:43 PM PST
